forex/views/index.html

```diff
@@ -18,10 +18,12 @@ <h1>Forex Exchange</h1>
         <div class="row">
             <div class="dropdowns">
                 <form>
-                    <input type="text" id="currencyDropdown1" list="json-datalist1" placeholder="Choose" class="padding-05">
+                    <input type="text" id="currencyDropdown1" list="json-datalist1" placeholder="Choose"
```

Comment:
In this part of the code, you are using` input` text fields for currency selection,Instead, it would be more appropriate to use `<select>` elements for dropdown functionality that includes the country names to follow the assessment